pkg/util/utils.go

```diff
@@ -200,3 +202,9 @@ func ConcatStrings(src []string, prefix string, postfix string) []string {
 	}
 	return ret
 }
+
+func OpenBrowser(urlString string) {
```

Comment:
This function doesn't add ta lot of value, so just remove it and inline the code instead. The only thing really duplicated is the error message. 

Also, we're trying to get rid of unscoped packages like the `util` package.